### PR TITLE
CET-8084 Adobe downgrade of backstage version 1.25 -> 1.22

### PR DESCRIPTION
**Pull request by igorRoog**
## Change description

Adobe asked for bumping backstage version, but unfortunately 1.25.2 was too newest for them so they asked for downgrade to 1.22.2. As backstage-plugin which uses 1.25.2 is already published, we won't downgrade it back to 1.22.2 but rather do one-off publish, with special version for Adobe.

## Type of change

- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [ ] Improvement (improves codebase without changing functionality)

### Development

- [ ] The changelog has been updated as appropriate
- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Review by FoodProduct — COMMENTED** [review]
Please fill out the PR description -- I have no context on this change 🤷‍♂️

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.6.5-a
```
Comment (FoodProduct) [inline CHANGELOG.md:3]: Could we have a more descriptive tag than `-a` please? 🙏

**Review by FoodProduct — COMMENTED** [review]
OK so the plan is not to merge this and instead manually run `publish`?

File: CHANGELOG.md (hunk 1 above)
Comment (cristinabuenahora) [reply CHANGELOG.md:3]: We purposely wanted to keep it vague -- hoping no one else pulls this

Comment (FoodProduct) [reply CHANGELOG.md:3]: Why not just call it `2.7.0-beta-1` or something then? 😝

**Pull request closed without merging**